### DISCUSSION TITLE
Welcome to InfantCareCompass Discussions!

### DISCUSSION THREAD

**Discussion by Amarjha01** [Amarjha01/InfantCareCompass#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by suvasis-patra** [comment]
Hi Amar, Suvasis this side. In your singup flow there is no checking if the user trying to singup already exist or not I want to implement that and there is no data validation send by the user I can implement input data validation using zod. I am willing to contribute as a Gssoc contributor
